language/move-prover/bytecode/tests/spec_instrumentation/opaque_call.exp

```diff
@@ -0,0 +1,240 @@
+============ initial translation from Move ================
+
+[variant baseline]
```

Comment:
What is the purpose of the variant attribute?

Reply:
After spec instrumentation, each function comes into variants, one for verification, one for inlining. ("baseline" is used for the inling)